src/components/Notification/styles.scss

```diff
@@ -55,6 +55,10 @@
   .notification-bar-label {
     margin-left: 10px;
   }
+
+  .notification-icon {
```

Comment:
Brukes denne noe sted?

Reply:
Ja, la den til for at ikonet ved feilmeldingen (rødt lite kryss) skulle sentreres i diven, og ikke være i toppen. Dermed er ikonet nå sentrert sammen med teksten og ikke et par pixler over

Reply:
👍